pkg/storage/ingest/reader.go

```diff
@@ -182,15 +182,15 @@ func (r *PartitionReader) stopDependencies() error {
 
 func (r *PartitionReader) run(ctx context.Context) error {
 	for ctx.Err() == nil {
-		r.processNextFetches(ctx)
+		r.processNextFetches(ctx, "running")
 	}
 
 	return nil
 }
 
-func (r *PartitionReader) processNextFetches(ctx context.Context) {
+func (r *PartitionReader) processNextFetches(ctx context.Context, state string) {
```

Comment:
Label is `phase` while variable is `state`. I would keep it consistent.

Reply:
I'm now passing `delayObserver prometheus.Observer` parameter.